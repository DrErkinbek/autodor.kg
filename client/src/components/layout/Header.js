import React from 'react';

const Header = () => {
    return(
        <>
        <nav className="blue darken-4" role="navigation">
            <div className="nav-wrapper container">
                <a id="logo-container" href="/" className="brand-logo">АВТОДОР</a>
            <ul className="right hide-on-med-and-down">
                <li><a href="/" className="yellow-text nav-link">Главная</a></li>
                <li><a href="/brands" className="yellow-text nav-link">Аккумуляторы</a></li>
                <li><a href="/tires" className="yellow-text nav-link">Шины</a></li>
                <li><a href="/disks" className="yellow-text nav-link">Диски</a></li>
                <li><a href="/accessories" className="yellow-text nav-link">Автоаксессуары</a></li>
            </ul>
            <ul id="nav-mobile" className="sidenav blue darken-4">
                <li><a href="/" className="yellow-text nav-link">Главная</a></li>
                <li><a href="/brands" className="yellow-text nav-link">Аккумуляторы</a></li>
                <li><a href="/tires" className="yellow-text nav-link">Шины</a></li>
                <li><a href="/disks" className="yellow-text nav-link">Диски</a></li>
                <li><a href="/accessories" className="yellow-text nav-link">Автоаксессуары</a></li>
            </ul>
            <a href="#!" data-target="nav-mobile" className="sidenav-trigger pulse"><i className="material-icons white-text">menu</i></a>
            </div>
        </nav>
        </>
    )
}

export default Header;