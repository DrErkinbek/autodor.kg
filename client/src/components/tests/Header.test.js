import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from '../layout/Header';

it('Header Component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
});

it('Header Component renders correctly and equal to snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
  
    expect(tree).toMatchSnapshot();
});