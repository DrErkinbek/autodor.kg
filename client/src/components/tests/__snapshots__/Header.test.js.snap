// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly and equal to snapshot 1`] = `
<nav
  className="blue darken-4"
  role="navigation"
>
  <div
    className="nav-wrapper container"
  >
    <a
      className="brand-logo"
      href="/"
      id="logo-container"
    >
      АВТОДОР
    </a>
    <ul
      className="right hide-on-med-and-down"
    >
      <li>
        <a
          className="yellow-text nav-link"
          href="/"
        >
          О компании
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/brands"
        >
          Аккумуляторы
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/tires"
        >
          Шины
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/disks"
        >
          Диски
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/accessories"
        >
          Автоаксессуары
        </a>
      </li>
    </ul>
    <ul
      className="sidenav blue darken-4"
      id="nav-mobile"
    >
      <li>
        <a
          className="yellow-text nav-link"
          href="/"
        >
          О компании
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/brands"
        >
          Аккумуляторы
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/tires"
        >
          Шины
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/disks"
        >
          Диски
        </a>
      </li>
      <li>
        <a
          className="yellow-text nav-link"
          href="/accessories"
        >
          Автоаксессуары
        </a>
      </li>
    </ul>
    <a
      className="sidenav-trigger pulse"
      data-target="nav-mobile"
      href="#!"
    >
      <i
        className="material-icons white-text"
      >
        menu
      </i>
    </a>
  </div>
</nav>
`;
